### PR TITLE
Change console style string to enum

### PR DESCRIPTION
**Pull request by sharifelgamal**

### CODE REVIEW

File: pkg/minikube/console/style_enum.go [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+	Kubectl
+	Meh
+	Embarrassed
+	Tip
+	Unmount
+	MountOptions
+	Fileserver
+)
+
+func (style StyleEnum) String() string {
```
Comment (tstromberg) [inline pkg/minikube/console/style_enum.go]: Any chance we could avoid implementing the String() function? I'm not clear who uses it.

**Comment by tstromberg** [comment]
@minikube-bot OK to test

File: pkg/minikube/console/style_enum.go (hunk 1 above)
Comment (sharifelgamal) [reply pkg/minikube/console/style_enum.go]: Done.

File: pkg/minikube/console/style.go [hunk 2]
```diff
@@ -47,67 +46,67 @@ type style struct {
...
+	Resetting:        {Prefix: "🔄  "},
+	DeletingHost:     {Prefix: "🔥  "},
+	Copying:          {Prefix: "✨  "},
+	Connectivity:     {Prefix: "📶  "},
+	Internet:         {Prefix: "🌐  "},
+	Mounting:         {Prefix: "📁  "},
+	Celebrate:        {Prefix: "🎉  "},
+	ContainerRuntime: {Prefix: "🎁  "},
+	Docker:           {Prefix: "🐳  "},
+	Crio:             {Prefix: "🎁  "}, // This should be a snow-flake, but the emoji has a strange width on macOS
```
Comment (tstromberg) [inline pkg/minikube/console/style.go]: This should be CRIO: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

File: pkg/minikube/console/style.go [hunk 3]
```diff
@@ -47,67 +46,67 @@ type style struct {
...
-	"verifying":         {Prefix: "🤔  "},
-	"verifying-noline":  {Prefix: "🤔  ", OmitNewline: true},
-	"kubectl":           {Prefix: "💗  "},
-	"meh":               {Prefix: "🙄  ", LowPrefix: lowWarning},
-	"embarrassed":       {Prefix: "🤦  ", LowPrefix: lowWarning},
-	"tip":               {Prefix: "💡  "},
-	"unmount":           {Prefix: "🔥  "},
-	"mount-options":     {Prefix: "💾  "},
-	"fileserver":        {Prefix: "🚀  ", OmitNewline: true},
+	IsoDownload:      {Prefix: "💿  "},
```
Comment (tstromberg) [inline pkg/minikube/console/style.go]: This should be ISODownload: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

**Pull request merged**